lib/utils.pm

```diff
@@ -1892,6 +1893,17 @@ sub install_patterns {
     }
 }
 
+=head2 wait_quit_zypper
+    wait_quit_zypper();
+
+This function wait zypper process to quit automatically.
+
+=cut
+
+sub wait_quit_zypper {
+    assert_script_run 'until ! pgrep zypper; do sleep 1; done';
+}
+
```

Comment:
lib/utils is already much bigger than 1k lines and hence hardly maintainable. I suggest to use a different or new module. What about `lib/zypper.pm` which includes `zypper_call` and all other "zypper" functions including your new one?

Reply:
Hi, Oliver, I understand what your meaning, while move all zypper functions to a new module need so much effort. I think it is better to do it later with a new PR. WDYT?

Reply:
Well, I would not have mentioned my suggestion if I would not have seen it realistic to implement :) If you think it's too much effort then you could just create a new library with just the new method. If you really want to keep it here then you could create a ticket for that plan. But I strongly suggest to plan such refactoring and improvement work as part of the "normal tickets", otherwise it's very likely for such tasks to be delayed for very long and never picked up.

Reply:
I just updated code and move the function to lib/zypper.pm, wait https://openqa.nue.suse.com/tests/5346088 and https://openqa.nue.suse.com/tests/5346092#details to verify this.
In fact, just move the method is ok for me, I mean much effort is for the old 'zypper' functions, it related so much things to do.